src/index.html

```diff
@@ -8,10 +8,187 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+        
+        <label for="surname">Surname:
+          <input 
+            type="text"
+            name="Surname"
+            id="surname"
+            autocomplete="off"
+            pattern="\w+"
+            class="input"
+            required
+          >
+        </label><br>
```

Comment:
```suggestion
        </label>
        <br>
```
New line for the new tag